src/els_command_ct_run_test.erl

```diff
@@ -0,0 +1,59 @@
+-module(els_command_ct_run_test).
+
+-export([ execute/1 ]).
+
+%% Invoked from the Common Test Hook
+-export([ publish_result/4, run_test/1, ct_run_test/1 ]).
+
+-include("erlang_ls.hrl").
+
+-spec execute(map()) -> ok.
+execute(#{ <<"module">>   := M
+         , <<"function">> := F
+         , <<"arity">>    := A
+         , <<"line">>     := Line
+         , <<"uri">>      := Uri
+         }) ->
+  Msg = io_lib:format("Running Common Test [mfa=~s:~s/~p]", [M, F, A]),
+  lager:info(Msg, []),
```

Comment:
Why this combination of `io_lib:format/2` + `lager:info/2` and not only use `lager:info/2`?
```suggestion
  lager:info("Running Common Test [mfa=~s:~s/~p]", [M, F, A]),
```

Reply:
Because the `Message` is re-used in the `Title`.